implementations/micrometer-registry-signalfx/src/main/java/io/micrometer/signalfx/SignalFxNamingConvention.java

```diff
@@ -77,25 +69,23 @@ public String name(String name, Meter.Type type, @Nullable String baseUnit) {
     // letters, numbers, underscores _ and hyphens - . This requirement is expressed in
     // the following regular expression:
     // ^[a-zA-Z][a-zA-Z0-9_-]*$
+    /**
+     * @see <a href=
+     * "https://docs.splunk.com/Observability/metrics-and-metadata/metric-names.html#dimension-names-and-value-standards">Dimension
```

Comment:
I think it'd be better to move this to the class-level JavaDoc and get rid of the old link that's currently there.